Mathlib/NumberTheory/Bernoulli.lean

```diff
@@ -341,7 +341,8 @@ theorem sum_range_pow (n p : ℕ) :
         mk fun p =>
           ∑ i ∈ range (p + 1), bernoulli i * (p + 1).choose i * (n : ℚ) ^ (p + 1 - i) / (p + 1)! by
       rw [← div_eq_iff (hne p), div_eq_mul_inv, sum_mul]
-      rw [PowerSeries.ext_iff] at this
+      -- Adaptation_note (2024-07-08): this was rw, but that created an instance goal
```

Comment:
We have the special syntax for adaptation notes now  (and I think the linter would have complained if not for the sneaky underscore)

Reply:
Fixed.